src/style.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: "avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.big_searchbar {
+  margin: 20px auto;
+}
+
+.search {
```

Comment:
You should add styles for focused state too, check it on figma
![image](https://user-images.githubusercontent.com/37161315/137140701-6b3cc780-1c50-4743-9424-b65b10d412fe.png)